### DISCUSSION TITLE
Authenticate does not work

### DISCUSSION THREAD

**Discussion by hauwm** [TumblThreeApp/TumblThree#338] [Q&A (Question & Answer)]
**Describe the bug**
I want to download hidden Tumblrs. Therefore I need to authenticate myself.

**To Reproduce**
1. Go to Settings - Connection
2. Click on 'Authenticate'

After a few seconds TumblThree terminates without any messages.

**Expected behavior**
Explorer window to open to enter username and password.

**Desktop (please complete the following information):**
 - TumblThree version: 1.3.0.96
 - OS: Windows 10 Pro
 - Browser: Edge
 - Version 89.0.774.54

**Comment by thomas694** [comment]
Please try, if the following helps.

> If you experience crashes right before or while logging in, it may be that on your system the "[Visual C++ Redistributable for Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48145)" is missing. You can download it from MS.

**Comment by hauwm** [comment]
Thank you, now it is working!
